Jenkinsfile

```diff
@@ -1,14 +1,69 @@
+def nexusSnapshotRepoDetails = getNexusSnapshotRepoDetails()
+def nexusReleaseRepoDetails = getNexusReleaseRepoDetails()
+def mavenSettingsId = getMavenSettingsId()
+def mavenAdditionalSettings = getMavenAdditionalSettings()
+def enceladusSlaveLabel = getEnceladusSlaveLabel()
+def deploymentStageRun = 'false'
```

Comment:
Are you sure we want this hardcoded value in the Jenkinsfile which we commit? I would do it the same way as the `enceladusSlaveLabel` and others.

Reply:
The reason for this particular value is it's used as a flag for the post step on whether to print out that there was no deployment done to Nexus.
I don't mind abstracting it if you guys would like but I feel it's not worth it.

Reply:
Value removed, resolving